docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc

```diff
@@ -215,6 +230,8 @@ Just for the domain or standalone
 }
 ----
 
+Please note that the operation"
```

Comment:
Leftover? It seems the sentence which is started by these words has been added above already.